x-pack/test_serverless/api_integration/test_suites/observability/apm_api_integration/feature_flags.ts

```diff
@@ -0,0 +1,223 @@
+    type: 'form-data',
+    params: {
+      body: {
+        service_name: 'uploading-test',
+        service_version: '1.0.0',
+        bundle_filepath: 'bar',
+        sourcemap: JSON.stringify(SAMPLE_SOURCEMAP),
+      },
+    },
+  });
+  return response.body;
+}
+
+export default function ({ getService }: APMFtrContextProvider) {
+  const apmApiClient = getService('apmApiClient');
+
+  describe('apm feature flags', () => {
+    describe('fleet migrations', () => {
+      it('rejects requests to save apm server schema', async () => {
+        try {
+          await apmApiClient.slsUser({
+            endpoint: 'POST /api/apm/fleet/apm_server_schema 2023-10-31',
+            params: {
+              body: {
+                schema: {
+                  tail_sampling_enabled: true,
+                },
+              },
+            },
+          });
+        } catch (err) {
+          expect(err.res.status).toBe(fleetMigrationResponse.statusCode);
+          expect(err.res.body).toStrictEqual(fleetMigrationResponse);
+        }
+      });
+
+      it('rejects requests to get unsupported apm server schema', async () => {
+        try {
+          await apmApiClient.slsUser({
+            endpoint: 'GET /internal/apm/fleet/apm_server_schema/unsupported',
+          });
+        } catch (err) {
+          expect(err.res.status).toBe(fleetMigrationResponse.statusCode);
+          expect(err.res.body).toStrictEqual(fleetMigrationResponse);
+        }
+      });
+
+      it('rejects requests to get migration check', async () => {
+        try {
+          await apmApiClient.slsUser({
+            endpoint: 'GET /internal/apm/fleet/migration_check',
+          });
+        } catch (err) {
+          expect(err.res.status).toBe(fleetMigrationResponse.statusCode);
+          expect(err.res.body).toStrictEqual(fleetMigrationResponse);
+        }
+      });
+    });
+
+    describe('agent configuration', () => {
+      it('rejects requests to get agent configurations', async () => {
+        try {
+          await apmApiClient.slsUser({
+            endpoint: 'GET /api/apm/settings/agent-configuration 2023-10-31',
+          });
+        } catch (err) {
+          expect(err.res.status).toBe(agentConfigurationResponse.statusCode);
+          expect(err.res.body).toStrictEqual(agentConfigurationResponse);
+        }
+      });
+
+      it('rejects requests to get single agent configuration', async () => {
+        try {
+          await apmApiClient.slsUser({
+            endpoint: 'GET /api/apm/settings/agent-configuration/view 2023-10-31',
+          });
+        } catch (err) {
+          expect(err.res.status).toBe(agentConfigurationResponse.statusCode);
+          expect(err.res.body).toStrictEqual(agentConfigurationResponse);
+        }
+      });
+
+      // Not possible to test the DELETE, as first we will need to get the service
+      xit('rejects requests to delete agent configuration', async () => {
```

Comment:
I believe you can create an agent configuration even if the service doesn't exist.

Reply:
but I guess the user won't have access

Reply:
@MiriamAparicio We cannot have skipped tests which have typing issues. This will break the build.
Either we completely comment out the test or delete them. Skipping is not an option if params are not provided correctly.

Same for test at L210

Also this user is a superuser, so you can try what Kate is recommending.